### PR TITLE
[ci] Add LUCI version of build-all for web

### PR DESCRIPTION
**Pull request by stuartmorgan**
Adds a new Linux_web configuration, and an initial test (build-all-packages) using it.

This is the simplest web test since it doesn't actually run anything, so starting with this one as a foundation to test future web-based tests.

Part of https://github.com/flutter/flutter/issues/114373

### CODE REVIEW

File: .ci.yaml [hunk 1]
```diff
@@ -12,6 +12,15 @@ platform_properties:
   linux:
     properties:
       os: Linux
+  linux_web:
+    properties:
+      dependencies: >-
+        [
+          {"dependency": "chrome_and_driver", "version": "version:114.0"}
+        ]
+      os: Ubuntu
```
Comment (stuartmorgan) [inline .ci.yaml:21]: Is there a preference on `Ubuntu` vs `Linux` for non-desktop tests?

File: .ci.yaml [hunk 2]
```diff
@@ -12,6 +12,15 @@ platform_properties:
...
+      cores: "8"
```
Comment (stuartmorgan) [inline .ci.yaml:22]: Out of curiosity, what's the default? I'm cargo-culting this, and don't know if we actually want it.

File: .ci.yaml (hunk 1 above)
Comment (keyonghan) [reply .ci.yaml:21]: Now everything is Ubuntu, and let's keep `Ubuntu` for any linux targets.

File: .ci.yaml (hunk 2 above)
Comment (keyonghan) [reply .ci.yaml:22]: There is no default, but different options like cores: 32. Unless the test needs a higher number of cores, we should use `8` as 32 cores bots are limited.

**Review by keyonghan — APPROVED** [review]
LGTM

**Pull request merged**